.github/workflows/quick-test.yml

```diff
@@ -25,6 +25,22 @@ jobs:
       - name: super-test
         run: |
             ./super-test.sh quick ${{ matrix.compiler }}
+  Linux-lock-tracking:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        compiler: [clang-9]
+        features: ["-DKJ_TRACK_LOCK_BLOCKING=1 -DKJ_SAVE_ACQUIRED_LOCK_INFO=1"]
+    steps:
+      - uses: actions/checkout@v2
+      - name: install dependencies
+        run: |
+            export DEBIAN_FRONTEND=noninteractive
+            sudo apt-get install -y build-essential git zlib1g-dev cmake libssl-dev ${{ matrix.compiler }}
+      - name: super-test
+        run: |
+            ./super-test.sh quick ${{ matrix.compiler }} cpp-features "${{matrix.features}}" extra-libs "-lrt"
```

Comment:
What is librt needed for?

Reply:
`timer_X` functions in the unit test. I was using it as a way to generate a signal that could run during a deadlock so I can test the TLS stuff.

Reply:
Fair enough, maybe worth a comment.